src/main/resources/org/sonar/plugins/checkstyle/rules.xml

```diff
@@ -1956,7 +1973,7 @@
     <param key="allowSamelineParameterizedAnnotation" type="BOOLEAN">
       <defaultValue>false</defaultValue>
     </param>
-    <param key="tokens" type="s[CLASS_DEF,INTERFACE_DEF,PACKAGE_DEF,ENUM_CONSTANT_DEF,ENUM_DEF,METHOD_DEF,CTOR_DEF,VARIABLE_DEF,PARAMETER_DEF,ANNOTATION_DEF,ANNOTATION_FIELD_DEF]">
+    <param key="tokens" type="s[CLASS_DEF,INTERFACE_DEF,PACKAGE_DEF,ENUM_CONSTANT_DEF,ENUM_DEF,METHOD_DEF,CTOR_DEF,VARIABLE_DEF,ANNOTATION_DEF,ANNOTATION_FIELD_DEF]">
```

Comment:
please prove that all existing instances of Rule that used such token will fail with exception, please share exception in issue, to let users quickly find migration notes by such exception.

Reply:
issue created: https://github.com/checkstyle/sonar-checkstyle/issues/228

Reply:
I did not mean new issue creation, just sharing exception with workaround in #222 should be enough

Reply:
ah ok. well, should I remove the issue and copy it into #222 or do we link #222 with #228 and close #228?

Reply:
Keep it, not a problem. Just do not create similar next time